doc/tutorials/pick_and_place_with_moveit_task_constructor/pick_and_place_with_moveit_task_constructor.rst

```diff
@@ -977,8 +946,8 @@ The second stage ("move_to_home") is a ``MoveTo`` type of stage. It inherits its
 
 In this case, we could tell that "move_to_home" was the root cause of the failure. The problem was a home state that was in collision. Defining a new, collision-free home position fixed the issue.
 
-Various hints
-^^^^^^^^^^^^^
+7.2 Stages
+^^^^^^^^^^^^^^^^^
```

Comment:
Nit: Why extend this line? I don't believe that the length of the underline matters as long as it's longer than the title, so this really doesn't affect anything, but I don't understand the intent of the change.

Reply:
Originally I added "7.2 " so I added the additional 4 characters to the underline but I forgot to fix it when I renamed the heading